indexer/services/vulcan/src/handlers/order-replace-handler.ts

```diff
@@ -0,0 +1,393 @@
+import { logger, runFuncWithTimingStat, stats } from '@dydxprotocol-indexer/base';
+import { createSubaccountWebsocketMessage, KafkaTopics } from '@dydxprotocol-indexer/kafka';
+import {
+  OrderFromDatabase,
+  OrderTable,
+  PerpetualMarketFromDatabase,
+  blockHeightRefresher,
+  perpetualMarketRefresher,
+  protocolTranslations,
+} from '@dydxprotocol-indexer/postgres';
+import {
+  CanceledOrdersCache,
+  OrderbookLevelsCache,
+  placeOrder,
+  PlaceOrderResult,
+  StatefulOrderUpdatesCache,
+  convertToRedisOrder,
+  removeOrder,
+  RemoveOrderResult,
+} from '@dydxprotocol-indexer/redis';
+import {
+  getOrderIdHash,
+  isLongTermOrder,
+  isStatefulOrder,
+  ORDER_FLAG_SHORT_TERM,
+  requiresImmediateExecution,
+} from '@dydxprotocol-indexer/v4-proto-parser';
+import {
+  IndexerOrder,
+  IndexerOrderId,
+  OffChainUpdateV1,
+  OrderPlaceV1_OrderPlacementStatus,
+  OrderReplaceV1,
+  OrderUpdateV1,
+  RedisOrder,
+} from '@dydxprotocol-indexer/v4-protos';
+import Big from 'big.js';
+import { IHeaders, Message } from 'kafkajs';
+
+import config from '../config';
+import { redisClient } from '../helpers/redis/redis-controller';
+import { sendMessageWrapper } from '../lib/send-message-helper';
+import { Handler } from './handler';
+
+/**
+ * Handler for OrderReplace messages.
```

Comment:
Add comment that this is only expected for vault orders currently.

Reply:
done!